src/System.Management.Automation/engine/Modules/NewModuleManifestCommand.cs

```diff
@@ -58,10 +58,10 @@ public object[] NestedModules
         private object[] _nestedModules;
 
         /// <summary>
-        /// Set the GUID in the manifest file.
+        /// Gets or sets the GUID in the manifest file.
         /// </summary>
         [Parameter]
-        public Guid Guid
+        public Guid GUID
```

Comment:
Please revert - we should use camel style. Also it is public API (breaking change).

Reply:
done